src/api/dutchx.ts

```diff
@@ -70,7 +70,7 @@ async function init(): Promise<DutchExchange> {
     amount: Balance,
     index: Index,
     userAccount: Account,
-  ) => dx.postSellOrder(t1, t2, index, amount, { from: userAccount, gasPrice: GAS_PRICE, gas: GAS_LIMIT_TESTING })
```

Comment:
This review is not for the current version, and it's just my opinion.
I find easier to use the old approach were u had a constant, but instead of using a single one (`GAS_LIMIT_TESTING`) we could define the gas limit per operation so it's more accurate.

Example in this case, could be sth like:
```js
 dx.postSellOrder(t1, t2, index, amount, { from: userAccount, gasPrice: GAS_PRICE, gas: GAS_LIMITS.postSellOrder }
```